aten/src/ATen/cuda/CUDAApplyUtils.cuh

```diff
@@ -620,23 +620,23 @@ bool CUDA_tensor_apply3(at::Tensor a,
     // Ignore overlaps when copying back; if we use THCTensor_copy
     // instead, it will recursively try and invoke ourselves to make
     // oldA contiguous.
-    oldA._copy_ignoring_overlaps_(a);
+    at::_copy_ignoring_overlaps_(oldA, a);
```

Comment:
This is breaking our convention. I don't think we've ever had inplace operations that are not methods. If you really want to change this, this should be `*_out`.

Reply:
`_copy_ignoring_overlaps_out` is fine with me